wg-notebooks/charter.md

```diff
@@ -0,0 +1,39 @@
+# WG Notebooks Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Notebooks is responsible for the user experience around Notebooks and other IDEs and their integrations with Kubeflow.
+The WG covers launching and managing different types of Notebooks on Kubeflow as well as workflows that Notebook extensions enable.
+
+### In scope
+
+#### Code, Web apps, Controllers and Services
+
+- APIs and tools used for launching Notebooks (e.g. Notebooks CRD/Controller).
+- Web UI to manage and connect to Notebook instances (e.g. Notebook Manager UI)
+- SDKs and integrations from inside the Notebooks (e.g. Git inside Notebooks, Kale)
+- Documentation for detailed description of using Notebooks in Kubeflow.
+
+#### Cross-cutting and Externally Facing Processes
+
+- Coordinating with Control Plane WG to ensure that Notebooks manifests are properly deployed with Kubeflow.
+- Coordinating with Central Dashboard WG to ensure the integration with the Notebook Manager UI is up-to-date.
+- Coordinating with release teams to ensure that the Notebooks features can be released properly.
```

Comment:
Releases of notebooks should be the responsibility of this WG.

Reply:
Added both an entry for this both in the artifacts owned by the wg as well as to the external facing processes.

Reply:
@kimwnasptd Maybe I'm missing the updated text can you point me at it?

I'm hung up on this line

>  Coordinating with release teams to ensure that the Notebooks features can be released properly.

I apologize for being pedantic but right now there is no clearly defined "release teams" for Kubeflow. So until such time as there is, the notebooks WG is responsible for releasing major/minor/patch releases on whatever cadence it decides for the artifacts it owns.

Reply:
@jlewi I was referring to these two lines:
https://github.com/kubeflow/community/pull/407/files#diff-f24ed473a8c33db922bc5532691b177bR26
https://github.com/kubeflow/community/pull/407/files#diff-f24ed473a8c33db922bc5532691b177bR18

I was under the impression that there would be a WG that would define what would go in a release, after communicating with the other WGs. Since the WG can set its own cadence then we can remove this sentence.

Reply:
Those links don't seem to point to anything relevant for me.

The process for determining what should be in Kubeflow 1.X.Y is TBD. The general thinking is that KF 1.X.Y would just take the most recent stable releases of each application. So each WG would be responsible for releasing artifacts and this would then be pulled into a future release. The WG could release artifacts on its own cadence which could be more frequent or less frequent then a KF release.

Reply:
@jlewi thanks for clarifying.

I'm still not able to understand what information you would like to be more clear/included in the charter. Do you think there should also be an explicit entry in the `In scope` section that will be specifying that this WG will be having its own release schedule every X months for releasing the artifacts of YZW?